src/main.rs

```diff
@@ -104,6 +104,10 @@ struct Opts {
     #[structopt(short = "V", long)]
     version: bool,
 
+    /// Print a backtrace even if the program ran successfully
+    #[structopt(long, env = "PROBE_RUN_BACKTRACE")]
+    force_backtrace: Option<String>,
```

Comment:
Is it useful to pass `--force-backtrace=false`? If not, using this would be simpler (you'd pass `--force-backtrace` to enable it):

```suggestion
    force_backtrace: bool,
```

Reply:
much nicer! done